### DISCUSSION TITLE
Ability to disable sprayers as server owner.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#6083] [Ideas and suggestions]
While the community is generally great, there are some who would choose to use the sprayers to graffiti hate symbols on the walls of the sub. Being able to disable sprayers to prevent this would be helpful as a server admin. ALTERNATIVELY, sprayers could be replaced with stencils, which can graffiti one of the symbols from the sub editor, with up to say three uses. This would still enable people to express themselves, while also limiting hate speech.

**Comment by PaladynMinak** [comment]
The sprayer purpose is to clean walls and/or color/mark them. I'm rather against disabling it, because game is constructed to allow players to mess up with things. So instead, in such situation someone could be ordered to clean walls from such signs and culprit punished in game, if it's role play wise, or even culprit could be ordered e.g. under guard, to remove his creations and additionally clean ballasts. Eventually, if it's just random griefer player, he could be moderated out of the server.
